src/markdownSyntaxHighlight.js

```diff
@@ -37,7 +37,7 @@ module.exports = function(str, language) {
       (highlights.isHighlightedAdd(j) ? " highlight-line-add" : "") +
       (highlights.isHighlightedRemove(j) ? " highlight-line-remove" : "") +
       "\">" +
-      line +
+      (line || '\n') + // https://github.com/11ty/eleventy-plugin-syntaxhighlight/pull/5
       "</div>";
```

Comment:
Another approach would be to unconditionally add the newline after each `</div>`. That might be cleaner. What do you think?

Reply:
Actually, that would require not using `<div>`s at all. Perhaps the cleanest solution is to change these `<div>`s into `<span>`s, and to add newlines consistently at the end of each line.